enterprise/server/invocation_stat_service/invocation_stat_service.go

```diff
@@ -195,13 +196,99 @@ func (i *InvocationStatService) GetTrend(ctx context.Context, req *inpb.GetTrend
 				return nil, err
 			}
 		}
-		rsp.TrendStat = append(rsp.TrendStat, stat)
+		res = append(res, stat)
+	}
```

Comment:
do you need to check `rows.Err()` after the loop here?

Reply:
Done.